src/main/java/net/earthcomputer/clientcommands/command/arguments/EnchantmentArgument.java

```diff
@@ -0,0 +1,39 @@
+package net.earthcomputer.clientcommands.command.arguments;
+
+import com.mojang.brigadier.StringReader;
+import com.mojang.brigadier.arguments.ArgumentType;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.suggestion.Suggestions;
+import com.mojang.brigadier.suggestion.SuggestionsBuilder;
+import net.minecraft.commands.SharedSuggestionProvider;
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.enchantment.Enchantment;
+
+import java.util.concurrent.CompletableFuture;
+
+public class EnchantmentArgument implements ArgumentType<Enchantment> {
```

Comment:
`ItemAndEnchantmentsPredicateArgument` left the chat

Reply:
I remember that `ItemAndEnchantmentsPredicateArgument` doesn't have enchantment suggestions

Reply:
no, and no.